### DISCUSSION TITLE
Best options for cloud deployment?

### DISCUSSION THREAD

**Discussion by BatuhanYilmaz26** [openai/whisper#510] [Q&A]
Hi, I've been building a Auto subtitled video generator. If you've deployed a whisper-powered application in the cloud, I'd like to know what your experience was like.
I've built the application using Streamlit, OpenAI's Whisper and ffmpeg. 
- Takes a YouTube link or a video file as input and generates a video with subtitles, transcript as .txt, .vtt, .srt files.
- Takes a video file and a transcript as .vtt or .srt file and generates a video with subtitles.
- Takes an audio file and generates a transcript as .txt, .vtt, .srt files.

Currently looking for collaborators with experience in flask/django, react and cloud deployment.
Let's collaborate if you're interested.

**Comment by agarwalprashant** [comment]
Hello @BatuhanYilmaz26,

I am interested in collaborating with you on your React and Flask development projects. I believe I can be of assistance and would like to help you in any way possible. Please let me know if you are open to working together. Thank you.

**Reply by BatuhanYilmaz26** [reply]
Hi @agarwalprashant,
I'm open to work together. Let's get in touch.

**Comment by SomeCodingUser** [comment]
I tried deploying it to AWS batch, but it wasn't fun. AWS Batch provides GPU support, but the configuration is not so straight forward if you're not used to AWS. Would also be up for a collaboration!

**Reply by BatuhanYilmaz26** [reply]
I had hard times with AWS as well.
Let's get in touch.

**Reply by SomeCodingUser** [reply]
I think it' cheapest and easiest to use the model on [replicate](https://replicate.com/openai/whisper/api#input-audio) but you'd still have to wrap it in custom code and host that somewhere. I'm not sure which would be the cheapest option, since the wrapper code would have to wait till the transcription is done which could take some minutes if the file is long and therefore it would increase cost if it's for example hosted on aws lambda or similar.
